File: services/internal-treasury/src/test/java/br/com/deltaglobalbank/domain/TestInternalTreasuryApplication.java
package br.com.deltaglobalbank.domain;

import br.com.deltaglobalbank.domain.internal_treasury.TestcontainersConfiguration;
import br.com.deltaglobalbank.internal_treasury.InternalTreasuryApplication;
import org.springframework.boot.SpringApplication;

public class TestInternalTreasuryApplication {

    public static void main(String[] args) {
        SpringApplication.from(InternalTreasuryApplication::main)
            .with(TestcontainersConfiguration.class)
            .run(args);
    }
}

File: services/internal-treasury/src/test/kotlin/br/com/deltaglobalbank/domain/TestInternalTreasuryApplication.kt
package br.com.deltaglobalbank.domain

import br.com.deltaglobalbank.domain.internal_treasury.TestcontainersConfiguration
import br.com.deltaglobalbank.internal_treasury.InternalTreasuryApplication
import org.springframework.boot.fromApplication
import org.springframework.boot.with


fun main(args: Array<String>) {
	fromApplication<InternalTreasuryApplication>().with(TestcontainersConfiguration::class).run(*args)
}
